Sources/FirebaseFirestore/Firestore+Swift.swift

```diff
@@ -35,3 +35,93 @@ extension Firestore {
     swift_firebase.swift_cxx_shims.firebase.firestore.firestore_collection(self, std.string(collectionPath))
   }
 }
+
+// An extension that adds the encoder and decoder functions required
+// to serialize and deserialize documents in Firebase. These are mostly
+// copy of the Swift implementation that's available within firebase-ios-sdk
+//
+public extension Firestore {
+    class Encoder {
+      public var dateEncodingStrategy: FirebaseDataEncoder.DateEncodingStrategy = .timestamp
+      public var dataEncodingStrategy: FirebaseDataEncoder.DataEncodingStrategy = .blob
+
+      public var nonConformingFloatEncodingStrategy: FirebaseDataEncoder
+        .NonConformingFloatEncodingStrategy = .throw
```

Comment:
This is a very confusing break, the type really should be kept as a joined token.

Reply:
Seems like just a typo, will fix